commitlint.config.js

```diff
@@ -1,6 +1,5 @@
 module.exports = {
     extends: [
-        '@commitlint/config-conventional',
-        '@commitlint/config-lerna-scopes',
```

Comment:
This only lets you use packages names (`near-api-js`) as commit scopes. @esaminu I found this a bit restrictive, i.e. I can't use `chore(release): ` for the Lerna release.

Reply:
hmm any way we can have defined scopes? I think scopes will be important as the mono repo scales for version resolution based on PR titles

Reply:
Scopes have no effect on versioning of packages - Lerna looks at the commit history and which files have been touched.